.eslintrc.yml

```diff
@@ -0,0 +1,51 @@
+extends: 'eslint:recommended'
```

Comment:
we have such file in other projects with eslint configuration, but not in this project. I think it would be good to have it here also.